config/container.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+return ['containerized' => true];
```

Comment:
This is temporary and should be removed, but was necessary to get logs up and running. 

Standard practice would be to log to `stderr` rather than files, and have the container runtime or the orchestrator direct the logs somewhere. However this [works poorly with php-fpm](https://buff.ly/2ANZTlI) as the logs are cluttered in the best syslog tradition and can't even be parsed. So we'll need to find another way to deal with logs, for example to write them locally as before and have a sidecar container upload them.

For the development environment, `stderr` should be enough however, so I should transform this into a agnostic configuration like `'logging.handlers' => 'stderr'`